pkgs/by-name/po/postfixadmin/package.nix

```diff
@@ -1,30 +1,48 @@
-{ fetchFromGitHub, stdenv, lib, nixosTests }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch2,
+  nixosTests,
 
-stdenv.mkDerivation rec {
+  configLocalPath ? "/etc/postfixadmin/config.local.php",
+  templatesCachePath ? "/var/cache/postfixadmin/templates_c",
```

Comment:
Why do we need those options? they are unused currently in the module and I doubt that we really need them.